src/elements/OcTextInput.vue

```diff
@@ -5,19 +5,24 @@
     :value="value"
     :placeholder="placeholder"
     :aria-label="placeholder"
-    @input="onInput($event.target.value)"
-    @focus="onFocus($event.target.value)"
+    ref="input"
+    @input="$_ocTextInput_onInput($event.target.value)"
+    @focus="$_ocTextInput_onFocus($event.target.value)"
+    @keydown="$_ocTextInput_onKeyDown($event)"
   />
 </template>
 
 <script>
+import OcButton from "./OcButton"
+
 /**
  * Form Inputs are used to allow users to provide text input when the expected
  * input is short. Form Input has a range of options and supports several text
  * formats including numbers. For longer input, use the form `Textarea` element.
  */
 export default {
   name: "oc-text-input",
+  components: { OcButton },
```

Comment:
no, not really

Reply:
I know - could not find an alternative approach
use a plain html button instead?

Reply:
um I don't think you need to reference it:
```diff
diff --git a/src/elements/OcTextInput.vue b/src/elements/OcTextInput.vue
index 8c85fc6..768e3b6 100644
--- a/src/elements/OcTextInput.vue
+++ b/src/elements/OcTextInput.vue
@@ -110,7 +110,7 @@ export default {
             <h3 class="uk-heading-divider">
                 Interactions
             </h3>
-            <button @click="_focus">Focus input below</button>
+            <oc-button text="Focus input below" @click="_focus" />
             <oc-text-input placeholder="Will you focus on me?" ref="inputForFocus"/>
         </section>
     </template>
```

works just fine for the button